config/templates/stack-template.json

```diff
@@ -17,19 +17,19 @@
           "{{$subnet.AvailabilityZone}}"
           {{end}}
         ],
-        "DesiredCapacity": "{{.WorkerCount}}",
+        "DesiredCapacity": "{{.WorkerASG.MinSize}}",
```

Comment:
OK so let's remove `DesiredCapacity` then 👍

Btw, I've noticed that omitting `DesiredCapacity` here would make the situation easier for #148 too.